tests/Frameworks/WordPress/Version_5_9/wp-config.php

```diff
@@ -37,7 +37,7 @@
 /** The database collate type. Don't change this if in doubt. */
 define( 'DB_COLLATE', '' );
 
-define('DISABLE_WP_CRON', 'true');
+define( 'DISABLE_WP_CRON', true ); // disable cron

```

Comment:
No, I don't think we have some notes anywhere on this (which was troublesome for me when writing the tests). I'll do it  and include notes about the custom plugin as well :+1:

Note: Cron is disabled to ensure we only get one trace from our request, otherwise, flakiness would have been introduced.